base/multidimensional.jl

```diff
@@ -797,8 +796,7 @@ end
 """
     accumulate!(op, B, A, dim=1)
 
-Cumulative operation `op` on `A` along a dimension, storing the result in `B`.
-The dimension defaults to 1. See also [`accumulate`](@ref).
+Cumulative operation `op` on `A` along a dimension, storing the result in `B`. See also [`accumulate`](@ref).
```

Comment:
Perhaps add a linebreak before the second sentence? We try to stay within 92 characters per line, see https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#code-formatting-guidelines